sass/responsive/_mobile.scss

```diff
@@ -16,8 +16,18 @@
                 border: none;
             }
         }
+        .modal {
+            .settings-modal {
+                &:not(.settings-modal--tabless) {
+                    &:not(.display--content) {
+                        .modal-content {
```

Comment:
Hopefully we can make this a lot cleaner and not so deeply nested in the future.